src/routers/handlers/confirm_company/index.ts

```diff
@@ -113,5 +115,10 @@ export class ConfirmCompanyHandler extends GenericHandler<ConfirmCompanyViewData
         }
         setPresenterAccountDetails(req, { ...presenterAccountDetails, companyName });
     }
+
+    private appendCompanyNumberQuery(url: String, req: Request) {
+        const companyNumber = this.getCompanyNumberFromRequest(req);
+        return url + "?" + QueryParameters.COMPANY_NUMBER + "=" + companyNumber;
```

Comment:
Instead concat with +, can we use like below?
return `${url}?${QueryParameters.COMPANY_NUMBER}=companyNumber`;